DigitalLearningSolutions.Web.Tests/Controllers/LearningMenu/LearningMenuControllerTests.cs

```diff
@@ -45,7 +47,7 @@ public void SetUp()
             }, "mock"));
             httpContextSession = new MockHttpContextSession();
 
-            controller = new LearningMenuController(logger, config, courseContentService, tutorialContentService, sessionService)
+            controller = new LearningMenuController(logger, config, courseContentService, sectionContentService, tutorialContentService, sessionService)

```

Comment:
Minor but this should probably be split over several lines as it's getting a bit long 🙂